### PR TITLE
Make neutron techspec compliant

### PR DESCRIPTION
**Pull request by epwn**

### CODE REVIEW

File: roles/neutron-common/tasks/main.yml [hunk 1]
```diff
@@ -108,3 +108,12 @@
...
     - logging
+
+- name: change neutron log file permissions
+  file: path={{ item }} mode=0644 owner=neutron group=neutron state=touch
+  with_items:
+    - /var/log/neutron/neutron-server.log
+    - /var/log/neutron/neutron-l3-agent.log
+    - /var/log/neutron/neutron-linuxbridge-agent.log
+    - /var/log/neutron/neutron-dhcp-agent.log
+    - /var/log/neutron/neutron-metadata-agent.log
```
Comment (j2sol) [inline roles/neutron-common/tasks/main.yml]: Does this need another entry for lbaas?

**Comment by j2sol** [comment]
ok to test

File: roles/neutron-common/tasks/main.yml (hunk 1 above)
Comment (epwn) [inline roles/neutron-common/tasks/main.yml]: lbaas log was not in my test environment.

Comment (j2sol) [inline roles/neutron-common/tasks/main.yml]: lbaas is added (optionally) with #1190  and specifically https://github.com/blueboxgroup/ursula/pull/1190/files#diff-dc497c30f745d0a73efc2f57e6e706a0R87

**Comment by j2sol** [comment]
retest

**Comment by j2sol** [comment]
This is no longer a question, lbaas log file needs to be addressed if lbaas is enabled.

**Comment by j2sol** [comment]
:+1: Good to merge once tests pass.

File: roles/neutron-common/tasks/main.yml [hunk 2]
```diff
@@ -108,3 +108,13 @@
...
+    - /var/log/neturon/neutron-lbaasv2-agent.log
```
Comment (j2sol) [inline roles/neutron-common/tasks/main.yml]: Typo. `neturon` -> `neutron`

**Comment by epwn** [comment]
Is this good to go now?

**Comment by j2sol** [comment]
LGTM.

**Pull request merged**